### PR TITLE
Add view for browsing and unfollowing followed hashtags

### PR DESCRIPTION
**Pull request by Tak**
Implements #2785

### CODE REVIEW

**Comment by connyduck** [comment]
Can you make slightly more advanced list items with an x button (like we have in "hidden domains")?
And instead of dialog before unfollow I'd make an "undo" snackbar (also like in "hidden domains").

**Comment by Tak** [comment]
Updated

File: app/src/main/java/com/keylesspalace/tusky/FollowedTagsActivity.kt [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                        ),
+                        Snackbar.LENGTH_SHORT
+                    )
+                        .show()
+                }
+            )
+        }
+    }
+
+    private suspend fun refreshDisplay() {
```
Comment (connyduck) [inline app/src/main/java/com/keylesspalace/tusky/FollowedTagsActivity.kt:97]: doesn't need to be suspending

**Review by connyduck — APPROVED** [review]
Will merge after Tusky 20 release

**Review by connyduck — CHANGES_REQUESTED** [review]
Sorry to un-approve this again, but I just found out that this api is paginated, so we need to implement that as well or it won't work when a lot of hashtags are followed https://docs.joinmastodon.org/methods/followed_tags/#get

I'd to it the same as in other places with `RemoteMediator`/`PagingSource`

**Comment by Tak** [comment]
Of course in 4.0.3, I'll rework it

**Comment by connyduck** [comment]
Default page size is 100, how many users follow that many hashtags... I'd be ok with merging it without pagination for now as well.

File: app/src/main/java/com/keylesspalace/tusky/components/followedtags/FollowedTagsViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import androidx.paging.cachedIn
+import com.keylesspalace.tusky.di.Injectable
+import com.keylesspalace.tusky.entity.HashTag
+import com.keylesspalace.tusky.network.MastodonApi
+import javax.inject.Inject
+
+class FollowedTagsViewModel @Inject constructor (
+    api: MastodonApi
+) : ViewModel(), Injectable {
+    val tags: MutableList<HashTag> = emptyList<HashTag>().toMutableList()
```
Comment (connyduck) [inline app/src/main/java/com/keylesspalace/tusky/components/followedtags/FollowedTagsViewModel.kt:17]:
```suggestion
    val tags: MutableList<HashTag> = mutableListOf()
```

**Review by connyduck — COMMENTED** [review]
It now crashes when opening the view while offline. I think you need to extend the try block in `FollowedTagsRemoteMediator`.

**Review by connyduck — APPROVED** [review]

**Pull request merged**